web/views/layout_view.ex

```diff
@@ -11,4 +11,14 @@ defmodule CoursePlanner.LayoutView do
   def get_program_name do
     Settings.get_value("PROGRAM_NAME")
   end
+
+  def render_user_role_based_navbar(%{assigns: %{current_user: current_user}} = conn) do
+    case current_user.role do
```

Comment:
why would this happen? every user should have a role, which is one of those, otherwise they won't have access